src/Stats/utils.js

```diff
@@ -1,9 +1,67 @@
+const moment = require('moment');
+
 const sortDataByDate = (dataList) => {
   const originalDataList = dataList.slice(0);
   const orderedDataListByDateFromNearToFar = originalDataList.sort((a, b) => (b.date - a.date));
   return orderedDataListByDateFromNearToFar;
 };
 
+const sumAmount = (dataList) => {
+  const result = dataList.reduce((sum, { amount, type }) => (type === 'Expense' ? sum - parseFloat(amount) : sum + parseFloat(amount)), 0);
+  return result > 0 ? `+$${result}` : `-$${0 - result}`;
+};
+
+const groupTransactionsByDate = (transactions) => {
+  const convertedSet = transactions.sort((a, b) => (b.date - a.date)).map((value) => ({
+    ...value,
+    amount: value.amount,
+    date: moment.unix(value.date).format('MMMM Do YYYY'),
+    type: value.type,
+  }));
+
+  const result = convertedSet.reduce((groups, transaction) => {
+    const myGroup = groups;
```

Comment:
```suggestion
    const myGroup = {...groups};
```
also myGroup is not a nice name